tests/sca_package/test_runner.py

```diff
@@ -113,6 +113,36 @@ def test_run_with_skip(mocker: MockerFixture, scan_result):
     assert record.check_id == "CKV_CVE_2020_29652"
 
 
+def test_prepare_and_scan(mocker: MockerFixture, scan_result):
```

Comment:
can you separate the test and make two out of it

Reply:
sure!